cmd/web3/start.go

```diff
@@ -87,33 +85,9 @@ func start(c *cli.Context) error {
 	}
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	err = cmd.Start()
+	err = cmd.Run()
 	if err != nil {
-		log.Fatalln("Starting command failed:", err)
-	}
-
-	done := make(chan error, 1)
-	go func() {
-		done <- cmd.Wait()
-	}()
-	// catch ctrl-c and kill
-	sigC := make(chan os.Signal, 2)
-	signal.Notify(sigC, os.Interrupt, syscall.SIGTERM)
```

Comment:
Why remove all these? We want to catch the no?

Reply:
It is already handled in main. This is duplicate code. https://github.com/gochain-io/web3/pull/55#discussion_r255975063

Reply:
This is required to pass the signal on to the sub process.

Reply:
From the [godoc](https://golang.org/pkg/os/exec/#CommandContext) for `exec.CommandContext`:

> The provided context is used to kill the process (by calling os.Process.Kill) if the context becomes done before the command completes on its own.